x/config/configurator_test.go

```diff
@@ -854,6 +1101,11 @@ func TestConfigurator(t *testing.T) {
 			tt := tc.test(t, mockCtrl)
 			cfg := New()
 
+			for key, value := range tt.env {
+				restore := setenv(t, key, value)
```

Comment:
It's the same but I like the explicit variable in this case. Evaluation order (the fact that `setenv(..)` is called right away) can be confusing for newcomers.